### ISSUE TITLE
Cant bypass CloudFare

### ISSUE THREAD

**Issue by van2kot** [keiyoushi/tachiyomi-extensions-source#4594]
### Source information

Infernal Void Scans 1.4.38

### Source language

English

### Steps to reproduce

1.Browse 
2.Sources 
3.Infernal Void Scans

### Expected behavior

Load mangas/manhwas

### Actual behavior

![Screenshot_20240813-161814_Tachiyomi](https://github.com/user-attachments/assets/ce17fa94-3b42-41f7-b9b5-0a0dc65f731d)

### Mihon/Tachiyomi version

Tachiyomi 0.15.3

### Android version

Android 11

### Other details

Tried to bypass cloudfare in web view too but it is not even loading 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by van2kot** [comment]
Only after trying all the cloudfare issues guide I have raised this complain

**Comment by Smol-Ame** [comment]
@van2kot you can pass Cloudflare through the extension. If you changed your user-agent, remember to restart the app & check WebView if it passes. If it loops, redo the steps.

But, from my quick check, seems like there's no point currently since all their manga pages/chapters are 404 (deleted/missing from their site).
